code/modules/shuttle/on_move.dm

```diff
@@ -324,6 +324,13 @@ All ShuttleMove procs go here
 
 /mob/living/afterShuttleMove(list/movement_force, shuttle_dir, shuttle_preferred_direction, move_dir)
 	. = ..()
+	if(movement_force["IGNOREBUCKLE"] && buckled && prob(50))//GET READY FOR A WILD RIDE
```

Comment:
feels like 50% is too high

Reply:
I think a 50/50 isn't that bad, but maybe drop it to 33%? 1 in 3 seems pretty reasonable.